Sources/ClientConfiguration.swift

```diff
@@ -10,7 +10,7 @@ import Foundation
 
 enum Defaults {
     static let cdaHost = "cdn.contentful.com"
-    static let locale = "en-US"
+//    static let locale = "en-US"
```

Comment:
yeah reverting that...i noticed it wasn't used. should probably just be deleted.